jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -844,3 +847,16 @@ function configure_kafka_jbpm_emitter(){
   fi
 
 }
+
+function configure_kie_executor_mdb(){
+  log_info "Configuring KieServerExecutorMfDB Max Sessions on ejb-jar.xml"
+  sed -i  's#<activation-config-property-value>queue/KIE.SERVER.EXECUTOR</activation-config-property-value>#<activation-config-property-value>queue/KIE.SERVER.EXECUTOR</activation-config-property-value>\
+          </activation-config-property>\
+          <activation-config-property>\
+          <activation-config-property-name>maxSession</activation-config-property-name>\
+          <activation-config-property-value>'${JBOSS_MDB_MAX_SESSIONS:-50}'</activation-config-property-value>#g' ${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/ejb-jar.xml
+
+  log_info "Configuring KieServerExecutorMDB Strict Max Pool"
+  sed -i  's#${jboss.mdb.strict.max.pool.size}#'${JBOSS_MDB_STRICT_MAX_POOL:-65}'#g' ${CONFIG_FILE}
```

Comment:
what's the point of doing it?
remove this env and do it as you did for the first time, just a system property on standalone.xml with its default file.

Reply:
Isn't replaced with behave test